src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3057,6 +3079,7 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
                 Debug.Assert(builder.InstanceInitializersForRecordDeclarationWithParameters is object);
 
                 var ctor = addCtor(builder.RecordDeclarationWithParameters);
+                builder.UpdateIsNullableEnabledForConstructorsAndFields(ctor.IsStatic, compilation, paramList);
```

Comment:
>builder.UpdateIsNullableEnabledForConstructorsAndFields(ctor.IsStatic, compilation, paramList); [](start = 16, length = 95)

We should also include base argument list, if present. This is part of the "body".